Tests/APITesters/ObjCAPITester/ObjCAPITester/RCEntitlementInfoAPI.m

```diff
@@ -31,7 +31,7 @@ + (void)checkAPI {
     RCPurchaseOwnershipType ot = [ri ownershipType];
     NSDictionary<NSString *, id> *rawData = [ri rawData];
 
-    if (@available(iOS 13.0, *)) {
+    if (@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *)) {
```

Comment:
The `tvOS` deployment target of API testers was set to iOS 14, so we weren't checking these.